src/interpreter/runtime/values/f32.js

```diff
@@ -101,3 +101,13 @@ export function createValue(value: f32): StackLocal {
     value
   };
 }
+
+export function createValueFromArrayBuffer(
```

Comment:
I'm not that keen on this pattern, but would like to get this merged, and refactor later - perhaps using a similar pattern to binop / unop